.github/workflows/build-and-test.yml

```
@@ -112,7 +112,7 @@ jobs:
       - name: Lint
         run: |
           make install-tools
-          make -j4 checklicense impi lint misspell
+          make -j4 for-all checklicense impi lint misspell
```

I think for this to work `CMD` needs to be set but that would require addressing missing targets in the examples or adding a go mod subgroup

```suggestion
          make -j4 for-all CMD="make checklicense impi lint misspell"
```